src/Std/Time/Duration.lean

```diff
@@ -0,0 +1,26 @@
+/-
+Copyright (c) 2024 Lean FRO, LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sofia Rodrigues
+-/
+prelude
+import Std.Time.DateTime
+
+namespace Std
+namespace Time
+open Internal
+
+set_option linter.all true
+
+/--
+Duration is just a period between two timestamps.
+-/
+def Duration := Timestamp
```

Comment:
This type seems to be a bit underdeveloped, I think that there should be at least the various arithmetic methods for things like adding a duration to a timestamp, and a way to print a duration.